File: CLASS 1/2920 음계.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    char ascending[] = "1 2 3 4 5 6 7 8";
    char descending[] = "8 7 6 5 4 3 2 1";

    char str[16];
    fgets(str, 16, stdin);

    if (strcmp(str, ascending) == 0)
    {
        printf("ascending");
    }
    else if (strcmp(str, descending) == 0)
    {
        printf("descending");
    }
    else
    {
        printf("mixed");
    }
    return (0);
}

File: CLASS 1/2920 음계.cs
using System;

namespace ConsoleApp1
{
    public class Program
    {
        static void Main()
        {
            string ascending = "1 2 3 4 5 6 7 8";
            string descending = "8 7 6 5 4 3 2 1";
            string input = Console.ReadLine();

            if( input == ascending)
            {
                Console.WriteLine("ascending");
            }
            else if( input == descending )
            {
                Console.WriteLine("descending");
            }
            else
            {
                Console.WriteLine("mixed");
            }
        }
    }
}
